<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MineSweeper</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res.qrc">
    <normaloff>:/res/flag.png</normaloff>:/res/flag.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="mainScrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>591</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="topBarWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>4800</width>
           <height>80</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="flagNumLabel">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>24</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 0, 0);</string>
            </property>
            <property name="text">
             <string>10</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_apprentice">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>35</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Apprentice</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_adept">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>35</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Adept</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btn_Restart">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>35</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../res.qrc">
              <normaloff>:/res/star.png</normaloff>:/res/star.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_expert">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>35</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Expert</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_custom">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>35</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Custom</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="clockLabel">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>24</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 0, 0);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QWidget" name="fieldWidget" native="true">
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>1000</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
